nodedata/nodedata.go

```diff
@@ -650,6 +657,21 @@ func (c *NodeData) checkOne(
 	return nodeData, hash, nil
 }
 
+func (c *NodeData) syncedToChainLogic(synced bool, lastNotSynced *time.Time, settings entities.ReportingSettings) bool {
```

Comment:
syncedToChainLogic. 

logic suffix is implicit everywhere. Maybe `isSyncedToChain` or `checkIfSyncedToChain`